src/cli.ts

```diff
@@ -78,26 +78,19 @@ program.command('docs')
     .action(generateTypedoc);
 
 /**
- * Unit tests command
+ * Jest commands
  */
-program.command('test:unit')
-    .description('Runs all unit tests for the extending project.')
-    .action(runUnitTests);
-
-/**
- * Integration tests command
- */
-program.command('test:int')
-    .description('Runs all integration tests for the extending project.')
+program.command('test <type>')
```

Comment:
Can you document the `type` arg? Guessing allowed values are `"unit"` and `"int"`?

Reply:
Is now better documented in the description